opm/parser/eclipse/EclipseState/Schedule/VFPProdTable.hpp

```diff
@@ -150,7 +148,11 @@ class VFPProdTable {
     }
 
     bool operator==(const VFPProdTable& data) const;
-    VFPProdTable& operator=(const VFPProdTable& data);
+
+    std::array<size_t,5> shape() const;
+
+    double operator()(size_t thp_idx, size_t wfr_idx, size_t gfr_idx, size_t alq_idx, size_t flo_idx) const;
+    double& operator()(size_t thp_idx, size_t wfr_idx, size_t gfr_idx, size_t alq_idx, size_t flo_idx);
```

Comment:
Could the mutating accessor be private?

Reply:
very likely it can, will try.

Reply:
yes, at the cost of a couple of const_casts.